spec/pubid_iec/identifier/create_spec.rb

```diff
@@ -32,6 +32,25 @@ module Pubid::Iec
           end
         end
       end
+
+      context "when the same stage applied to different types" do
+        context "TR" do
+          let(:params) { { type: "TR", stage: "40.99" } }
+
+          it "renders stage" do
+            expect(subject.to_s).to eq("IEC ADTR #{number}")
+          end
+        end
+      end
+
+      context "TS" do
+        let(:params) { { type: "TS", stage: "40.99" } }
+
+        it "renders stage" do
+          expect(subject.to_s).to eq("IEC ADTS #{number}")
+        end
+      end
```

Comment:
@ronaldtse is these tests is not enough?

Reply:
Can you add for DISH and ISH? Thanks.

Reply:
@ronaldtse I didn't implement ISH and DISH identifiers parsing. I was asking for examples here: https://github.com/metanorma/pubid-iec/issues/115#issuecomment-1612336819, but didn't get answer yet.
I can just implement it to parse identifiers like `IEC TR DECDISH 123`, but I have feelings we don't have identifiers like this.

Reply:
The document type is “ISH” (interpretation sheet), “TR” is technical report, so they cannot be set together.

Reply:
@ronaldtse added ISH type and tests for it, but I cannot find information on "DISH" typed stage (stage codes) nor examples of identifiers.
The same for "FDIS" typed stage.